component/http/middleware/middleware.go

```diff
@@ -51,12 +55,6 @@ type responseWriter struct {
 	writer              http.ResponseWriter
 }
 
-var (
```

Comment:
Variables should be after types IIRC, but since we do not have a linter for it I think it can wait to be changed back.

Reply:
Usually I keep the struct and the `methods` together. And in the variables there is no usage of the type.
Is there a standard? Can you point me to it?
The only thing that I usually do is constants first, vars after (except if a var uses a type), then structs close together with the methods so that they are spread out through the file.

Reply:
Looks good! I also like that style; nevermind what I said, I had read it somewhere but the standard library does as you do: https://cs.opensource.google/go/go/+/refs/tags/go1.17.7:src/net/http/http.go;l=133